src/core/xfa/xhtml.js

```diff
@@ -65,99 +141,163 @@ function checkStyle(style) {
     .join(";");
 }
 
-class A extends XmlObject {
+const NoWhites = new Set(["body", "html"]);
```

Comment:
This should be `NoWhites` -> `NoWhitespace` instead, right?

(I'd suggest just folding this change into a pending XFA-patch, to reduce unnecessary code churn.)